tests/test.h

```diff
@@ -674,6 +674,15 @@ void test_consumer_verify_assignment0(const char *func,
         test_consumer_verify_assignment0(__FUNCTION__, __LINE__, rk,           \
                                          fail_immediately, __VA_ARGS__)
 
+
+void test_consumer_assign_by_rebalance_protocol(
```

Comment:
We can put these two new functions after `test_consumer_incremental_unassign` in the header and in the implementation as they're in different order